neo/Prompt/CommandBase.py

```diff
@@ -0,0 +1,130 @@
+from abc import ABC, abstractmethod
+from neo.Prompt.Utils import get_arg
+from typing import List
+
+
+class ParameterDesc():
+    def __init__(self, name, description, optional=False):
+        self.name = name
```

Comment:
Please add doc strings like we did for `CommandDesc` something like

```
name: 1 word parameter identifier
description: short description of the purpose of the parameter. What does it configure/do
optional: flag indicating whether the parameter is optional. Defaults to mandatory.
```
in google style format